### PR TITLE
fix(sequencer): fix config

### PR DESCRIPTION
**Pull request by noot**
## Summary

`ASTRIA_SEQUENCER` should be `ASTRIA_SEQUENCER_` - also small doc updates

## Testing
i ran it

### CODE REVIEW

**Review by SuperFluffy — APPROVED** [review]
We should have a test for this.

**Review by sambukowski — APPROVED** [review]

File: crates/astria-sequencer/src/service/mempool.rs [hunk 1]
```diff
@@ -74,7 +74,18 @@ fn handle_check_tx(req: request::CheckTx) -> response::CheckTx {
             };
         }
     };
-    let signed_tx = SignedTransaction::try_from_raw(raw_signed_tx).unwrap();
```
Comment (SuperFluffy) [inline crates/astria-sequencer/src/service/mempool.rs:77]: This is really orthogonal to the rest...

Comment (noot) [reply crates/astria-sequencer/src/service/mempool.rs:77]: should i add to a different PR? sorry i was just running some tests and also found this :p

Comment (SuperFluffy) [reply crates/astria-sequencer/src/service/mempool.rs:77]: Yeah, that would be much cleaner. 🙏

Comment (noot) [reply crates/astria-sequencer/src/service/mempool.rs]: https://github.com/astriaorg/astria/pull/418
